PrivateHeaders/MobileCoreServices/LSApplicationWorkspace.h

```diff
@@ -0,0 +1,143 @@
+//
+//  LSApplicationWorkspace.h
+//  WebDriverAgent
+//
+//  Created by Jonathan Beyrak-Lev on 08/07/2019.
+//
+
+/* Generated by RuntimeBrowser
+ Image: /System/Library/Frameworks/MobileCoreServices.framework/MobileCoreServices
+ */
+
+@interface LSApplicationWorkspace : NSObject {
+  NSXPCConnection * _connection;
+  NSMutableDictionary * _createdInstallProgresses;
+//  LSInstallProgressDelegate * _delegateProxy;
```

Comment:
please remove the commented stuff if we don't need it

Reply:
Done